### PR TITLE
docs(downstream): add additional low-code tools

### PR DESCRIPTION
**Pull request by adnanrahic**
https://deploy-preview-4897--cubejs-docs-staging.netlify.app/config/downstream

### CODE REVIEW

File: docs/content/Configuration/Downstream/Bubble.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  />
+</div>
+
+### Self-hosted Cube
+
+For a Cube instance publicly available at a specific `HOST`, the REST API URL would be `HOST/cubejs-api/v1`. Please refer to the [REST API page](https://cube.dev/docs/rest-api) for details.
+
+You will also need to generate a JSON Web Token that would be used to authenticate requests to Cube.
+
+Please check the [Security page](https://cube.dev/docs/security#generating-json-web-tokens-jwt) to learn how to generate a token. We suggest generating a long-lived JWT that won't expire anytime soon.
```
Comment (rpaik) [inline docs/content/Configuration/Downstream/Bubble.mdx:35]:
```suggestion
Please check the [Security page](https://cube.dev/docs/security#generating-json-web-tokens-jwt) to learn how to generate a token. We suggest generating a long-lived JWT that won't expire soon.
```

Comment (rpaik) [reply docs/content/Configuration/Downstream/Bubble.mdx:35]: I think we can remove the word "anytime" (and same probably applies below)

File: docs/content/Configuration/Downstream/Bubble.mdx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Display the data in Bubble
+
+Bubble has an amazing feature where you can drag and drop UI components into the design dashboard.
+
+Once you've added a chart, you configure the data points. More specifically, the series categories and series data. For the series categories and series data you need to select the `index` and `OrdersByMonth` state followed by the data point you want to use in order to populate the chart.
```
Comment (rpaik) [inline docs/content/Configuration/Downstream/Bubble.mdx:116]:
```suggestion
Once you added a chart, you configure the data points. More specifically, the series categories and series data. For the series categories and series data you need to select the `index` and `OrdersByMonth` state followed by the data point you want to use in order to populate the chart.
```

**Comment by rpaik** [comment]
@adnanrahic a couple of minor nits 😄

**Comment by adnanrahic** [comment]
I'll do a final tidy-up, and merge today. 👌

**Pull request merged**